server/reducer.js

```diff
@@ -6,20 +6,10 @@ function setState(state, newState) {
   return Object.assign({}, state, newState);
 }
 
-function addOrder(state, order) {
```

Comment:
I'm starting to wonder if this is an anti-pattern to have only one catch-all reducer.. working okay so far but smells a little.. what do you think?